stale-projects.js

```diff
@@ -1,5 +1,5 @@
 exports.getStaleProjects = (currentProjects, targetProjects) => (
-  currentProjects.filter((p) => !targetProjects.includes(p.node.project.name))
+  currentProjects.filter((p) => !targetProjects.includes(p.name))
```

Comment:
also here some confusing naming: `targetProjects` appears to be a list of project _names_, while `currentProjects` appears to be list of _projects_